### ISSUE TITLE
Frida not started on 4.4 android

### ISSUE THREAD

**Issue by maxben14** [frida/frida#1459]
frida -U -f by.advasoft.android.troika.app -l 5.js --no-pause
     ____
    / _  |   Frida 12.11.12 - A world-class dynamic instrumentation toolkit
   | (_| |
    > _  |   Commands:
   /_/ |_|       help      -> Displays the help system
   . . . .       object?   -> Display information about 'object'
   . . . .       exit/quit -> Exit
   . . . .
   . . . .   More info at https://www.frida.re/docs/home/
Failed to spawn: java.lang.SecurityException: Not allowed to start activity Intent { act=android.intent.action.MAIN cat=
[android.intent.category.LAUNCHER] flg=0x10000000 pkg=by.advasoft.android.troika.app cmp=com.my.appTest }

**Comment by k7eon** [comment]
hello, did u tried to run app and after that without "-f flag"?
"frida -U by.advasoft.android.troika.app -l 5.js --no-pause"

**Comment by maxben14** [comment]
Yes, after run app i can attach frida, but me need on start app do attach frida. App on start do interesting operations and i want hook this.

**Comment by Minxin** [comment]
I had the same problem, use this:
adb forward tcp:27042 tcp:27042
adb forward tcp:27043 tcp:27043
it works ,maybe frida won't automatically forward the port on old version android.
